### PR TITLE
feat: add 篝火营地

### PR DESCRIPTION
**Pull request by sintak**
enjoy!

### CODE REVIEW

File: lib/routes/gouhuo/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        url: `https://gouhuo.qq.com/content/getHomeMainContent?tabId=${tabId}`,
+        headers: {
+            Referer: `https://gouhuo.qq.com/`,
+        },
+    });
+    const list = response.data.data;
+
+    const title = await cache.getTitle(cache, ctx);
+    const tabName = await cache.getTabName(cache, ctx, tabId);
+    const description = await cache.getDescription(cache, ctx);
```
Comment (DIYgod) [inline lib/routes/gouhuo/index.js]: 这里要调用三次 _getProfile，读取三次缓存，很浪费，只调用一次 _getProfile 就能取到所有数据

**Comment by rubbenchan** [comment]
Can't work now.Please fix it! ;)

**Comment by sintak** [comment]
Will fix that shortly.

**Comment by sintak** [comment]
Fixed in [#4148](https://github.com/DIYgod/RSSHub/pull/4148)

**Pull request merged**